lib/quibble.mjs

```diff
@@ -1,122 +1,103 @@
-import quibble from './quibble.js'
-
-export default quibble
-export const reset = quibble.reset
-export const ignoreCallsFromThisFile = quibble.ignoreCallsFromThisFile
-export const config = quibble.config
-export const isLoaderLoaded = quibble.isLoaderLoaded
-
-function loadQuibble () {
-  global.__quibble = { stubModuleGeneration: 1 }
-}
-
-export function globalPreload () {
-  return `(${loadQuibble.toString()})()`
-}
-
-export function getGlobalPreloadCode () {
-  return `(${loadQuibble.toString()})()`
-}
-
-export async function resolve (specifier, context, nextResolve) {
-  const resolve = () => nextResolve(
-    specifier.includes('__quibble')
-      ? specifier.replace('?__quibbleresolvepath', '').replace('?__quibbleoriginal', '')
-      : (specifier.match(/^[a-zA-Z]:/) ? `file://${specifier}` : specifier),
-    context
-  )
-
-  if (specifier.includes('__quibbleresolvepath')) {
-    const resolvedPath = new URL((await resolve()).url).pathname
-    const error = new Error()
-    error.code = 'QUIBBLE_RESOLVED_PATH'
-    error.resolvedPath = resolvedPath
-    throw error
+import quibble from "./quibble.js";
+
+export default quibble;
+export const reset = quibble.reset;
+export const ignoreCallsFromThisFile = quibble.ignoreCallsFromThisFile;
+export const config = quibble.config;
+export const isLoaderLoaded = quibble.isLoaderLoaded;
+
+let __quibble;
+
+export async function resolve(specifier, context, nextResolve) {
+  const resolve = () =>
+    nextResolve(
+      specifier.includes("__quibble")
+        ? specifier
+            .replace("?__quibbleresolvepath", "")
+            .replace("?__quibbleoriginal", "")
+        : specifier,
+      context
+    );
+
+  if (specifier.includes("__quibbleresolvepath")) {
+    const resolvedPath = new URL((await resolve()).url).pathname;
+    const error = new Error();
+    error.code = "QUIBBLE_RESOLVED_PATH";
+    error.resolvedPath = resolvedPath;
+    throw error;
   }
 
-  if (!global.__quibble.quibbledModules) {
-    return resolve()
+  if (!__quibble.quibbledModules) {
+    return resolve();
   }
 
-  if (specifier === 'quibble') {
-    return resolve()
+  if (specifier === "quibble") {
+    return resolve();
```

Comment:
Quibble uses standard, right? So I'd assume these should remain `'` and semicolon-free? (Maybe it's not configured to lint the .mjs file?)

Reply:
Hmm... probably!

(any chance I can get you to adopt `prettier`, which is practically the standard in the JavaScript world today)?)

Reply:
(will fix the lint problem with mjs-s)

Reply:
Probably not going to switch, sorry! Almost every package I own is on StandardJS and I'm not active enough to switch them all / churn